pkgs/tools/security/gnome-keysign/default.nix

```diff
@@ -53,7 +53,7 @@ python3.pkgs.buildPythonApplication rec {
     gst_all_1.gstreamer
     gst_all_1.gst-plugins-base
     (gst_all_1.gst-plugins-good.override { gtkSupport = true; })
-    gst_all_1.gst-plugins-bad # for zbar plug-in
+    (gst_all_1.gst-plugins-bad.override { enableZbar = true; })# for zbar plug-in
```

Comment:
```suggestion
    (gst_all_1.gst-plugins-bad.override { enableZbar = true; }) # for zbar plug-in
```

Reply:
I also thought about formatting the expression with `nixpkgs-fmt`, but that changed quite a lot.
Shall I still do that?

Reply:
I think I already formatted it. Using `nixpkgs-fmt` master I do not see any changes in this expression.

Reply:
I think I already formatted it. Using `nixpkgs-fmt` master I do not see any changes in this expression.

Reply:
@jtojnar Well, I do using nixpkgs-fmt 0.6.1